content/hi/automation.md

```diff
@@ -0,0 +1,12 @@
+---
+टाइटल: ऑटोमेशन 
+---
+## ऑटोमेशन का क्या मतलब है ?
+ऑटोमेशन का अर्थ स्वचालन हैं । यह एक तकनीक है जिसमे किसी भी मशीन को फुल्ली आटोमेटिक बनाया जाता हैं आसान भाषा में बताया जाए तो ये एक लेबर सेविंग टेक्नोलॉजी हैं जिसमे किसी भी काम को करने के लिए कम से कम लोगो की आवशकता होती हैं, उद्धहरण के लिए जैसे : चैटबॉट , जब भी आप कोई वेबसाइट पर विजिट करते है तो आपके पास एक हेलो पॉपअप होता है तो वो एक ऑटोमेशन है | आजकल ऑटोमेशन हर क्षेत्र में प्रयोग किया जाता है | 
+
+## ऑटोमेशन क्या संकट का समाधान लाता हैं ?
+ऑटोमेशन प्रबंधनो और ऑपरेटरों को एनवायरनमेंट मैनेज यानी की पुराने मैन्युअल ऑपरेशन्स को ऑटोमेट करने में मदद करता है | उद्धहरण के तोर पर हम चैटबॉट समझाते है - पहले वेबसाइट के मालिकों को बहुत से अधिकारियों को काम पर रखना पड़ता था जो की सिर्फ वेबसाइट पर आने वाले लोगो की तकलीफे सुने , पर ऑटोमेशन के जरिये ये काम आसान होगया है - जिससे की प्रोडक्ट की कॉस्ट में भी गिरावट हुई है | 
+
+## ये कैसे मदद करता है ?
+ऑटोमेशन एनवीरोमेंट्स के प्रबंधन और विन्यास के कार्य को बहुत तेजी से करता है - जिससे की कम इंजीनियरिंग समय लगता है | जैसे की चैटबॉट ने कार्य को ऑटोमेट करलिया अब बॉट कुछ जरूरी सवाल लोगो से पूछेगा - जैसे की यूजर की डिटेल्स, और उस सवाल का जवाब जो यूजर ने बहुत बार पूछा है, इससे हमे ज़्यादा अधिकारिओं की ज़रुरत नहीं पड़ेगी , और गलतियों के चान्सेस भी कम होंगे | 

```

Comment:
- एनवीरोमेंट्स is wrong, should be एनवायरनमेंट्स
- लोगो is wrong, should be लोगों
- The last sentence should be framed as 'गलतियों की संभावना भी कम होंगी'